packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -372,8 +378,10 @@ describe("single-choice question", () => {
 
             it("should open the cross-out menu when focused and spacebar pressed", async () => {
                 // Arrange
+                jest.setTimeout(10000);
```

Comment:
Is this something we really need? A 10 second timeout?

Reply:
I received timeout errors from this test multiple times originally. I tested it again just now and it seems to be okay without, so I took it out. Will keep an eye on it though.

Reply:
Was able to take it out without a problem!